.github/workflows/ci.yml

```diff
@@ -15,6 +15,8 @@ env:
   PIP_DISABLE_PIP_VERSION_CHECK: "1"
   PIP_NO_PYTHON_VERSION_WARNING: "1"
   PYTHON_LATEST: "3.11"
+  # avoid warnings about shallow checkout, but also make sure pytest accepts us.
+  SETUPTOOLS_SCM_PRETEND_VERSION: "99.9.9"
```

Comment:
Is this related to hatch? I know this is specific to `setuptools-scm`. Does hatch use it?

Reply:
Ah, I see, it's a wrapper.